reports/gas-used-output.txt

```diff
@@ -5,47 +5,47 @@
 ·······················|······················|············|··············|·············|···············|··············
 |  Contract            ·  Method              ·  Min       ·  Max         ·  Avg        ·  # calls      ·  eur (avg)  │
```

Comment:
Captain Europe for the rescue! 

Seriously though, if we enforce it, let's use USD, or hide the column altogether if it is not filled if possible.

Also I am not sure about how useful this information is without extra context. Like, `handleOps` is more or less useless  as it may accept whatever operations we make it. What do we learn from it?

Reply:
our current gas checks are not too useful. maybe the minimum of handleOps.
This one becomes useful once #93  is merged.